guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -183,6 +183,79 @@
 
     },
 
+    "IMPORT": {
+        
+        "ACTION_ACKNOWLEDGE"          : "@:APP.ACTION_ACKNOWLEDGE",
+
+        "BUTTON_CANCEL": "Cancel",
+        "BUTTON_CLEAR": "Clear",
+        "BUTTON_IMPORT": "Import Connections",
+
+        "CONNECTIONS_IMPORTED_SUCCESS": "{NUMBER} connections imported successfully.",
```

Comment:
You'll need to use pluralization here so that "connection" vs. "connections" is chosen appropriately based on the value of `NUMBER`.